src/constants/transactions.js

```diff
@@ -2,8 +2,15 @@ const maxMessageLength = 64;
 const minAccountBalance = 5000000;
 const minFeePerByte = 1000;
 
+const DUMMY_RECIPIENT_ADDRESS = 'lskdxc4ta5j43jp9ro3f8zqbxta9fn6jwzjucw7yt';
+const DEFAULT_NUMBER_OF_SIGNATURES = 1;
+const DEFAULT_SIGNATURE_BYTE_SIZE = 64;
+
 export {
   maxMessageLength,
```

Comment:
This a mix of camelCase and UPPERCASE constants. We didn't have uppercase constants in the past, they spread gradually. Given you prefer them, would you be so kind and change the rest too. If not in this PR, then please just create a ticket so we do it later.

Reply:
done: https://github.com/LiskHQ/lisk-desktop/issues/3437